scripts/scripts.js

```diff
@@ -58,48 +58,101 @@ export function decorateTooltipAndModalLinks(main) {
   });
 }
 
+function scheduleProcessing(processor, options = { chunkTime: 5 }) {
+  let lastTime = 0;
+  const processChunk = (timestamp) => {
+    const deadline = {
+      didTimeout: false,
+      timeRemaining: () => Math.max(0, options.chunkTime - (timestamp - lastTime)),
+    };
+    lastTime = timestamp;
+
+    if (processor(deadline)) {
+      requestAnimationFrame(processChunk);
+    }
+  };
```

Comment:
When I see this call, I don't think the `scheduleProcessing` method adds much value… it seems like just a wrapper that calls `requestAnimationFrame` twice.

At a high level, I think I'd approach this like:
1. Get a list of all elements that need to be processed
2. Create a generic `batchProcess(items, batchSize, processFn)` that creates batches of `batchSize` items and calls the `processFn` with each batch
3. Have the `processFn` do the processing for the given array of items and trigger the `requestAnimationFrame` (or other deferring API) as needed

That way 2. will offer a generic API we can re-use to batch process essentially anything, and the consumer can decide which deferring API to use depending on the use case (i.e. `setTimeout` for high priority background work, `requestIdleCallback` for low priority background work, `requestAnimationFrame` for DOM operations, etc.)